system/hardware/tici/hardware.py

```diff
@@ -96,6 +96,9 @@ def get_sound_card_online(self):
     return (os.path.isfile('/proc/asound/card0/state') and
             open('/proc/asound/card0/state').read().strip() == 'ONLINE')
 
+  def get_sound_input_device(self):
+    return "(hw:0,0)"
```

Comment:
is this necessary? looks like this relies on some sounddevice naming scheme?

Reply:
I think this is from ALSA. I remember it crashed when I didn't specify a device but will test again without